src/server/supplements/har/harTracer.ts

```diff
@@ -252,8 +252,9 @@ export class HarTracer {
       status: response.status(),
       statusText: response.statusText(),
       httpVersion: response.httpVersion(),
-      cookies: cookiesForHar(response.headerValue('set-cookie'), '\n'),
-      headers: response.headers().map(header => ({ name: header.name, value: header.value })),
+      // These are bad values that will be overwritten bellow.
+      cookies: [],
+      headers: [],
```

Comment:
I am afraid this will break tracing because we do not wait for the barriers. Can we bring best-effort headers back?

Reply:
Let's sort it out on the tracing side?